encoding/rs/encode.go

```diff
@@ -61,6 +62,26 @@ func (g *Encoder) Encode(inputFr []fr.Element) ([]Frame, []uint32, error) {
 	return frames, indices, nil
 }
 
+// PadPolyEval pads the input polynomial coefficients to match the number of evaluations
+// required by the encoder.
+func (g *Encoder) PadPolyEval(coeffs []fr.Element) ([]fr.Element, error) {
+	if len(coeffs) > int(g.NumEvaluations()) {
+		return nil, fmt.Errorf("the provided encoding parameters are not sufficient for the size of the data input")
+	}
+
+	pdCoeffs := make([]fr.Element, g.NumEvaluations())
```

Comment:
could be more efficient if you allocate and copy
https://github.com/consensys/gnark-crypto/blob/v0.13.0/ecc/bn254/fr/element.go#L193